src/Annotation/EloquaAppCloudInteractiveResponder.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Drupal\eloqua_app_cloud\Annotation;
+
+use Drupal\Component\Annotation\Plugin;
+
+/**
+ * Defines a Eloqua AppCloud Interactive Responder
+ * as a base for other plugin types.
+ *
+ * @see \Drupal\eloqua_app_cloud\Plugin\EloquaAppCloudInteractiveResponderBase
+ * @see plugin_api
+ *
+ * @Annotation
+ */
+class EloquaAppCloudInteractiveResponder extends Plugin {
+
+  /**
+   * The plugin ID.
+   *
+   * @var string
+   */
+  public $id;
+
+  /**
+   * The label of the plugin.
+   *
+   * @var \Drupal\Core\Annotation\Translation
+   *
+   * @ingroup plugin_translatable
+   */
+  public $label;
+
+  /**
+   * The list of fields required by the plugin.
+   *
+   * These must be presented in the form:
+   * fieldList = {
+   *   "EmailAddress" = "{{Contact.Field(C_EmailAddress)}}"
+   *  },
+   *
+   * The key (i.e. EmailAddress), is arbitrary, but must be used consistently and is case sensitive.
+   * The value (i.e. Contact.Field(C_EmailAddress)), is the internal Eloqua field description.
+   * Extracting this value from Eloqua
+   * (possibly by using a call to https://secure.p01.eloqua.com/API/bulk/2.0/contacts/fields )
+   * is left as an exercise for the reader.
+   *
+   * @var array
+   *
+   */
+  public $fieldList;
+
+  /**
+   * The API type (contacts or customObject).
+   *
+   * @var string
+   */
+  public $api;
+
+
+  /**
+   * The name of the queue worker this plugin requires.
+   *
+   * @var string
+   */
+  public $queueWorker;
+
+  /**
+   * The tye of response this plugin expects (either synchronous or asynchronous).
+   *
+   * @var string
+   */
+  public $respond;
+
+  /**
+   * The description of this plugin. It will be returned to Eloqua on update requests.
+   *
+   * @var string
+   */
+  public $description;
+
+  /**
+   * Really a boolean flag but defined as a string since Eloqua seems to need
+   * lowercase "true" and "false".
+   *
+   * @var string
```

Comment:
Same comment as previously: can we enforce an actual boolean here on our own interface, but "translate" it to a string for Eloqua somewhere in the controller?

Reply:
Everything in an annotation is a string, so I am not sure of the value of converting a string to a boolean, returning the boolean, and then converting it back to a string. 
How about if I just remove my internal musing from the comment? 
To whit: 
// Eloqua requires a lowercase "true" or "false".

Reply:
I thought you could defined booleans in an annotation? Just remove the quotes around "true" or "false"

Reply:
Yep: https://www.drupal.org/docs/8/api/plugin-api/annotations-based-plugins

Reply:
I really did look for annotation types documentation on api.d I really did (eyeroll).

Even that embarrassing oversight aside, I feel weird about declaring a boolean when the end consumer wants a string, but don't feel strongly about it.